waiting_room_test.go

```diff
@@ -160,6 +168,14 @@ var waitingRoomRule = WaitingRoomRule{
 	LastUpdated: &testTimestampWaitingRoom,
 }
 
+var waitingRoomSettings = WaitingRoomSettings{
+	SearchEngineCrawlerBypass: true,
+}
+
+var waitingRoomSettingsPatch = WaitingRoomSettingsPatch{
+	SearchEngineCrawlerBypass: &t,
```

Comment:
to save yourself doing this, the library does have a `BoolPtr` helper you can lean on for easier type conversion.